frontend/app/src/types/user.ts

```diff
@@ -9,12 +9,12 @@ import { LedgerActionEnum } from '@/types/history/ledger-action/ledger-actions-t
 import { ModuleEnum } from '@/types/modules';
 import { PriceOracleEnum } from '@/types/price-oracle';
 import { type ToSnakeCase } from '@/types/common';
-import { snakeCaseTransformer } from '@/services/axios-tranformers';
+import { camelCaseTransformer } from '@/services/axios-tranformers';
 
 const OtherSettings = z.object({
   krakenAccountType: KrakenAccountType.optional(),
   frontendSettings: z.string().transform(arg => {
-    const data = arg ? snakeCaseTransformer(JSON.parse(arg)) : {};
+    const data = arg ? camelCaseTransformer(JSON.parse(arg)) : {};
     return FrontendSettings.parse(data);
   }),
```

Comment:
hmmm, don't we have some unit tests that cover this scenario?

Reply:
the test we have doesn't test this part :(
I'll update the test